api/src/wfl/api/workloads.clj

```diff
@@ -100,8 +101,8 @@
   (try
     (start-workload! tx (create-workload! tx workload-request))
     (catch Throwable cause
-      (throw (ex-info "Error executing workload request"
-                      {:workload-request workload-request} cause)))))
+      (throw (ex-info (str "Error executing workload request - " (.getMessage cause))
+                      (ex-data cause) cause)))))
```

Comment:
The issue was that exceptions were being wrapped and didn't have the same `:type`.

Reply:
I think the wrapping can be removed. From your example log, the wrapping is not even adding context, just noise.

Reply:
I'll remove the wrapping everywhere it doesn't add context--in a few places that wrapping is where we add in the request or workload or something, which I think is probably pretty useful for debugging